org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/resource/AbstractApiExposedFeature.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.resource;
+
+import java.util.Set;
+
+import jakarta.ws.rs.container.DynamicFeature;
+import jakarta.ws.rs.container.ResourceInfo;
+import jakarta.ws.rs.core.FeatureContext;
+
+import org.eclipse.scout.rt.api.data.ApiExposeHelper;
+import org.eclipse.scout.rt.api.data.ApiExposed;
+import org.eclipse.scout.rt.platform.BEANS;
+import org.eclipse.scout.rt.platform.util.ObjectUtility;
+import org.eclipse.scout.rt.rest.RestApplication;
+import org.eclipse.scout.rt.rest.RestApplicationScope;
+import org.eclipse.scout.rt.rest.RestApplicationScopes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractApiExposedFeature implements DynamicFeature {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AbstractApiExposedFeature.class);
+
+  @Override
+  public void configure(ResourceInfo resourceInfo, FeatureContext context) {
+    ApiExposed annotation = getApiExposedAnnotation(resourceInfo, context);
+
+    if (annotation == null && !isDefaultRestrict()) {
+      LOG.debug("ApiExposed annotation is not present, no {} installed for {}", ApiExposedFilter.class.getSimpleName(), resourceInfo);
+    }
+    else if ((annotation == null && isDefaultRestrict()) || !annotation.value()) {
+      // compute if operation is *not* api-exposed, if *not* exposed then install filter to restrict access
```

Comment:
"compute" is already done when we reach this line of code. remove "compute" in this javadoc?